test/unit/adapters/http.js

```
@@ -29,6 +29,64 @@ describe('supports http with nodejs', function () {
     }
   });
 
+  it('should NOT parse the timeout property', function (done) {
```

Can this test rather be something like `should throw an error if the timeout property is not parsable as a number`

---

Done! :)